.pre-commit-config.yaml

```diff
@@ -1,10 +1,21 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v1.4.0
-    hooks:
-    - id: flake8
-      additional_dependencies: [flake8-docstrings, flake8-quotes, flake8-annotations, flake8-import-order]
-    - id: check-added-large-files
-    - id: detect-private-key
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v1.4.0
+  hooks:
+  - id: check-added-large-files
+  - id: detect-private-key
+  - id: trailing-whitespace
+  - id: end-of-file-fixer
+- repo: https://github.com/psf/black
+  rev: 23.7.0
+  hooks:
+    - id: black
+      language_version: python3.11
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  # Ruff version.
+  rev: v0.0.280
```

Comment:
might as well bump this up to 292 while we're here

Reply:
Ok I didn't know if we wanted to be consistent across our normalizers. I can go bump

Reply:
Yeah, idk, I kinda go back and forth. I don't know if there's a particularly good way to handle it, especially since Ruff and Black both get new releases/features so often. I wish we could have ChatGPT auto increment them across all of our repos at once.